### DISCUSSION TITLE
Empty login screen after installing version 1.9.10

### DISCUSSION THREAD

**Discussion by xaver-ius** [craigk5n/webcalendar#447] [Q&A] [answered]
I was lucky to install the package using sqlite, but after saving the settings the login screen is empty.
I don't have an idea what has to be done ...
To see the installation: https://hdf-baustelle.org/install/ with pw: 789456
Here you can have access: ssh.strato.de:22
user: sftp_helper@hdf-baustelle.org
pw: to be send to an email adress

**Comment by max951** [comment, marked as answer]
If you have the possibility to change the php.ini, put there  
`display_errors = On.`
This will display the reason for the error.

If you have a German browser, you can also try changing the translations/German.txt file.
Try out whether the change of the line
`charset: =`
to
`charset: utf-8`
helps.

**Reply by xaver-ius** [reply]
I have no access to php.ini, but charset helped! 
Thank you so much, max951
